stable/prometheus-operator/values.yaml

```diff
@@ -275,6 +275,19 @@ alertmanager:
     ##
     podAntiAffinityTopologyKey: kubernetes.io/hostname
 
+    ## Assign the alertmanager instance to run on specific nodes
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    ##
+    affinity: {}
+    # requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
I believe you miss to add `nodeAffinity:` before this line:
```
  # nodeAffinity:
        # requiredDuringSchedulingIgnoredDuringExecution:
...
```

Reply:
Thanks for that. Probably a copy/paste error.